### PR TITLE
Add a Dockerfile together with an example

### PR DESCRIPTION
**Pull request by denderello**
This will add the Dockerfile required to build the Docker image together with an usage example.

### CODE REVIEW

**Comment by denderello** [comment]
ping @giantswarm/team-core @zeisss @teemow

File: README.md [hunk 1]
```diff
@@ -12,7 +12,14 @@ Clone the git repository: https://github.com/giantswarm/docker-cloudflare-cli.gi
 
 ## Running docker-cloudflare-cli
 
-TODO: Add an example.
+You can run the `cfcli` command via the Docker container with bind mounting a
```
Comment (hectorj2f) [inline README.md]: I think it should be better `within a docker container`

Comment (denderello) [inline README.md]: right, fixed it.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,14 @@ Clone the git repository: https://github.com/giantswarm/docker-cloudflare-cli.gi
...
+`.cfcli.yml` to `/root/.cfcli.yml`:
+```
+docker run --rm -it \
+  -v ~/.cfcli.yml:/root/.cfcli.yml
```
Comment (hectorj2f) [inline README.md]: missing `\` at the end of the line

**Comment by hectorj2f** [comment]
LGTM

**Pull request merged**